packages/contracts-bedrock/test/L1/OptimismPortal2.t.sol

```diff
@@ -32,6 +37,12 @@ contract OptimismPortal2_Test is CommonTest {
         super.enableFaultProofs();
         super.setUp();
 
+        // zero out contracts that should not be used
+        assembly {
+            sstore(l2OutputOracle.slot, 0)
+            sstore(optimismPortal.slot, 0)
+        }
```

Comment:
Good idea, haven't seen this pattern before

Reply:
Thanks, we will want to delete this in the future, its just a sanity check to guarantee that we don't use the wrong contracts in any test